portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,6 +14,10 @@
 
 package com.google.sps.servlets;
 
+import static com.google.sps.servlets.Constants.cType;
+import static com.google.sps.servlets.Constants.encoding;
+import static com.google.sps.servlets.Constants.says;
```

Comment:
There is no need to use static import each constant. You can import the constants file once.